codalab/worker/bundle_state.py

```diff
@@ -76,23 +76,27 @@ def __init__(
         self.is_anonymous = is_anonymous
         self.metadata = metadata
         self.args = args
-        self.dependencies = {
-            DependencyKey(dep["parent_uuid"], dep["parent_path"]): Dependency(
-                parent_name=dep["parent_name"],
-                parent_path=dep["parent_path"],
-                parent_uuid=dep["parent_uuid"],
-                child_path=dep["child_path"],
-                child_uuid=dep["child_uuid"],
-                location=dep.get("location", None),
+        self.dependencies = [
+            (
+                DependencyKey(dep["parent_uuid"], dep["parent_path"]),
```

Comment:
Don't need the key if it's a list because it's already in `Dependency`

Reply:
That's true. I have updated the code.